templates/worker/01-worker-container-runtime/_base/files/policy.yaml

```diff
@@ -0,0 +1,19 @@
+filesystem: "root"
+mode: 0644
+path: "/etc/containers/policy.json"
+contents:
+  inline: |
+    {
+        "default": [
+            {
+                "type": "insecureAcceptAnything"
+            }
+        ],
+        "transports":
+            {
+                "docker-daemon":
```

Comment:
`updatePolicyJSON` does not create this entry. Does it need to? (I don’t think so, unless there are users of Docker left around.) If not, it seems a bit unexpected to me that the template is non-trivially different from configs generated by `updatePolicyJSON`.

Reply:
This is the default template that s laid down by the MCO at start up. I looked at what a fresh cluster has for policy.json and just used that for the template her.
When we want to add something to policy.json, the MCO uses this template and adds to it.